syntax = "proto3";

import "graph.proto";

package proto_transport_router;

message RouterSettings {
	int32 wait_time = 1;
	double velocity = 2;
}

message StopIdByName {
	string name = 1;
	uint32 id = 2;
}

message TransportRouter {
	RouterSettings settings = 1;
	proto_graph.Graph graph = 2;
	proto_graph.Router router = 3;
	repeated StopIdByName stop_id_by_name = 4;
}